W-14996918-Release-Studio-7-17-0-Docs-CP (#518)
mulesoft/docs-studio: modules/ROOT/pages/update-modules.adoc

= Update Your Modules

> Every time Studio starts up it checks for updated modules in your open projects. You can choose to update to the latest minor version of your module or upgrade directly to the latest version available. Upgrading to a major version of a module can introduce compatibility issues in your project. Review the release notes of your modules before upgrading to a major version.

Reviewer: ```suggestion
Every time Studio starts, it checks for updated modules in your open projects. You can update to the latest minor version of your module or upgrade directly to the latest version available. Upgrading to a major version of a module can introduce compatibility issues in your project. Review the release notes for your modules before upgrading to a major version.
```

Author: OK! Thanks!

> NOTE: If you xref:change-jdk-config-in-projects.adoc[select Java 17 for your project], Studio can notify you about updating your modules to make them compatible with the selected Java version.

Reviewer 2: ```suggestion
NOTE: If you xref:change-jdk-config-in-projects.adoc[select Java 17 for your project], Studio automatically searches Exchange and suggests the modules and connectors to upgrade to make your app compatible with Java 17.
```

Author: OK! Thanks!

To manually check for module updates:

. Ensure that you are logged in to Anypoint Platform.

> . In the menu bar, select the *Manage Mule Modules* (image:manage-mule-modules-icon.png[2%,2%]) button.

Reviewer: ```suggestion
. In the menu bar, click *Manage Mule Modules* (image:manage-mule-modules-icon.png[2%,2%]).
```

Author: Sorry! I read it but I didn't see it! :/ 
Thanks!

+
image::manage-mule-modules-location.png[]

> NOTE: This feature does not support modules that are inherited from a parent pom file.

Reviewer 2: ```suggestion
NOTE: This feature does not support modules that are inherited from a parent POM file.
```

Author: OK! Thanks!

You can also manage all dependencies in your project:

> . In the menu bar, select the *Manage dependencies* icon image:manage-dependencies.png[20,20].

Reviewer: Is the icon a button? If so, use click.

Author: Yes, it is! Thanks!

Studio updates your selected dependencies.

This process works if you manage your dependencies using Studio. 

> To use Maven properties to specify your modules version:

Reviewer: ```suggestion
To use Maven properties to specify your module version:
```

Author: OK! Thanks!
